src/Blazored.Video/BlazoredVideo.razor.Properties.cs

```diff
@@ -0,0 +1,215 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+using Blazored.Video.Support;
+using Microsoft.JSInterop;
+
+namespace Blazored.Video
+{
+	public partial class BlazoredVideo
+	{
+		public bool Autoplay
+		{
+			get { return GetValue<bool>(); }
+			set { SetValue(value); }
+		}
+
+		public object AudioTracks
+		{
+			get { throw new NotSupportedException("No Browser supports this api yet"); }
+		}
+
+		public TimeRanges[] Buffered
+		{
+			get { throw new NotImplementedException("The Buffered Property is not yet implemented"); }
+		}
+
+		public class TimeRanges
+		{
+			public int Length { get; set; }
+
+			public int Start(int index)
+			{
+				throw new NotImplementedException();
+			}
+
+			public int End(int index)
+			{
+				throw new NotImplementedException();
+			}
+		}
+
+		public bool Controls
+		{
+			get { return GetValue<bool>(); }
+			set { SetValue(value); }
+		}
+
+		public string CurrentSrc
+		{
+			get { return GetValue<string>(); }
+			set { SetValue(value); }
+		}
+
+		public int CurrentTime
+		{
+			get { return GetValue<int>(); }
+			set { SetValue(value); }
+		}
+
+		public bool DefaultMuted
```

Comment:
According to MDN "This property has no dynamic effect." - I would prefer not to include setters on properties that are non-functional